Made improvements to GitHub pages landing page.

nurse-the-code opened the pull request:
Testing instructions:

Pull this commit and open up `index.html` as a static file in a web browser. Does everything look good? Does it look good in different browsers? On different screen sizes?

## Before

| Desktop width   | Mobile width   |
|-----------------|---------------|
| <img width="1820" alt="Screenshot 2024-12-31 at 14 12 11" src="https://github.com/user-attachments/assets/a1e892cb-e47e-4b53-b7d5-b6d5ce05cbc0" /> | <img width="568" alt="Screenshot 2024-12-31 at 14 12 33" src="https://github.com/user-attachments/assets/eab53425-cd28-420d-bb18-d11e5bea5fd3" /> |

## After

| Desktop width   | Mobile width   |
|-----------------|---------------|
| <img width="1820" alt="Screenshot 2024-12-31 at 14 11 24" src="https://github.com/user-attachments/assets/be140c0c-1584-493b-94be-c9e11668f794" /> | <img width="612" alt="Screenshot 2024-12-31 at 14 15 10" src="https://github.com/user-attachments/assets/7616b3b1-103e-4439-bee3-4aafe4b6a3e8" /> |

nurse-the-code committed:
Made improvements to GitHub pages landing page.

nurse-the-code commented:
@yezr Should I add hyperlinks to the logos? If so where should each one point?

yezr commented:
> @yezr Should I add hyperlinks to the logos? If so where should each one point?

Sure! RCVRC logo to the main page and RCTab logo to the RCTab page of the website.

yezr commented:
looks good to me!

nurse-the-code committed:
Added hyperlinks to RCVRC and RCTab logos.

nurse-the-code committed:
Ran Prettier on index.html and styles/style.css

nurse-the-code committed:
Updated copyright.

nurse-the-code committed:
Fixed name of copyright holder

nurse-the-code commented:
I added/fixed a couple things:

- added hyperlinks
- re-formatted `index.html` and `styles/style.css` with [Prettier](https://prettier.io/)
- updated copyright year
- fixed copyright holder name (let me know if it also needs to include BrightSpots)

Otherwise, ready to merge on your approval.

yezr commented:
LGTM! merge away!
